tests/outputs/correct_pdf_output.txt

```diff
@@ -1,137 +1,193 @@
-| 7.7.1. OVERVIEW | 68 |
-| 7.7.2. INDEX REVIEW | 68 |
-| 7.7.3. ONGOING MAINTENANCE | 68 |
-
-| 7.8 STOXX INDUSTRY AND SUPERSECTOR LEGACY INDICES | 70 |
-|---------------------------------------------------|-----|
-| 7.8.1. OVERVIEW | 70 |
-| 7.8.2. INDEX REVIEW | 71 |
-| 7.8.3. ONGOING MAINTENANCE | 71 |
-
-| 7.9 EURO STOXX SUPERSECTOR 5/10/40 INDICES | 72 |
-|---------------------------------------------|-----|
-| 7.9.1. OVERVIEW | 72 |
-| 7.9.2. INDEX REVIEW | 72 |
-| 7.9.3. ONGOING MAINTENANCE | 73 |
-
-| 7.10 STOXX EUROPE 600 INDUSTRY 30-15 INDICES | 74 |
-|----------------------------------------------|-----|
-| 7.10.1. OVERVIEW | 74 |
-| 7.10.2. INDEX REVIEW | 74 |
-| 7.10.3. ONGOING MAINTENANCE | 75 |
-
-| 7.11. STOXX SEMICONDUCTOR 30 INDEX | 76 |
-|-------------------------------------|-----|
-| 7.11.1. OVERVIEW | 76 |
-| 7.11.2. INDEX REVIEW | 76 |
-| 7.11.3. ONGOING MAINTENANCE | 77 |
-
-## 8. STOXX EQUAL WEIGHT INDICES
-
-| 8.1. STOXX EQUAL WEIGHT INDICES | 78 |
-|--------------------------------|-----|
-| 8.1.1. OVERVIEW | 78 |
-| 8.1.2. INDEX REVIEW | 78 |
-| 8.1.3. ONGOING MAINTENANCE | 78 |
-
-## 9. STOXX BLUE-CHIP INDICES
-
-| 9.1 STOXX GLOBAL AND COUNTRY BLUE-CHIP INDICES | 80 |
-|------------------------------------------------|-----|
-| 9.1.1. OVERVIEW | 80 |
-| 9.1.2. INDEX REVIEW | 81 |
-| 9.1.3. ONGOING MAINTENANCE | 84 |
-
-| 9.2 EURO STOXX 50 | 85 |
-|-------------------|-----|
-| 9.2.1. OVERVIEW | 85 |
-| 9.2.2. INDEX REVIEW | 85 |
-| 9.2.3. ONGOING MAINTENANCE | 86 |
-
-| 9.3 STOXX REGIONAL BLUE-CHIP INDICES | 88 |
-|-------------------------------------|-----|
-| 9.3.1. OVERVIEW | 88 |
-| 9.3.2. INDEX REVIEW | 88 |
-| 9.3.3. ONGOING MAINTENANCE | 89 |
-
-| 9.4 STOXX GLOBAL 150 | 91 |
-|----------------------|-----|
-| 9.4.1. OVERVIEW | 91 |
-| 9.4.2. INDEX REVIEW | 91 |
-| 9.4.3. ONGOING MAINTENANCE | 91 |
-
-| 9.5 STOXX BALKAN 50 EQUAL WEIGHT | 92 |
-|-----------------------------------|-----|
-| 9.5.1. OVERVIEW | 92 |
-| 9.5.2. INDEX REVIEW | 92 |
-| 9.5.3. ONGOING MAINTENANCE | 93 |
-
-| 9.6 STOXX CANADA 60 | 94 |
-|---------------------|-----|
-| 9.6.1. OVERVIEW | 94 |
-| 9.6.2. INDEX REVIEW | 94 |
-| 9.6.3. ONGOING MAINTENANCE | 95 |
-
-## 10. STOXX DIVIDEND INDICES
-
-| 10.1 STOXX SELECT DIVIDEND INDICES | 96 |
-|-----------------------------------|-----|
-| 10.1.1. OVERVIEW | 96 |
-| 10.1.2. INDEX REVIEW | 96 |
-| 10.1.3. STOXX SELECT DIVIDEND INDICES | 99 |
-| 10.1.4. ONGOING MAINTENANCE | 101 |
-
-| 10.2 STOXX ASEAN-FIVE SELECT DIVIDEND 50 | 104 |
-|---------------------------------------------|-----|
-| 10.2.1. OVERVIEW | 104 |
-| 10.2.2. INDEX REVIEW | 104 |
-| 10.2.3. ONGOING MAINTENANCE | 105 |
-
-| 10.3 STOXX ASEAN SELECT DIVIDEND 30 | 106 |
-|-------------------------------------|-----|
-
-STOXX logo with text "Part of DEUTSCHE BÖRSE GROUP"
\ No newline at end of file
+STOXX INDEX METHODOLOGY GUIDE
+
+
+
+# CONTENTS
+
+6.5.1. OVERVIEW
+49
+7.11. STOXX SEMICONDUCTOR 30 INDEX
+76
+6.5.2. INDEX REVIEW
```

Comment:
It looks like the original model follow the the proper layout to output the left column first and then the right. However, the new model does not proper follow the layout?

Reply:
solved with the sgl fix